src/main/java/gregtech/integration/GroovyScriptCompat.java

```diff
@@ -222,12 +223,12 @@ public static String getGroovyItemString(GTRecipeInput recipeInput) {
                 builder.append(")");
             }
 
-            /*if (itemStack.serializeNBT().hasKey("tag")) {
```

Comment:
I expect this was commented out for reason.

Also having CT stuff in GroovyScript seems wrong. That is kind of coupling that should not be created.

@brachy84 can you please take a look at this as you do GroovyScript stuff.

Reply:
Yes, commented it because CT is not compatible with GrS and GrS had no conversion method at that point.